zkevm-circuits/src/circuit_tools/memory.rs

```diff
@@ -70,7 +87,18 @@ impl<F: Field, C: CellType> Memory<F, C> {
     ) {
         for bank in self.banks.iter() {
             bank.build_constraints(cb, is_first_row.expr());
-            // cb.generate_lookup_table_checks(bank.tag());
+        }
+    }
+
+    pub(crate) fn build_lookups(&self, meta: &mut ConstraintSystem<F>) {
```

Comment:
Generate on MemoryBank level?

Reply:
Any particular reason why?

Reply:
Potentially support different memory types that have their own logic.